.github/workflows/debos.yml

```diff
@@ -0,0 +1,89 @@
+name: Build debos recipe
+
+on:
+  # run on pull requests to the main branch if debos recipes are changed
+  pull_request:
+    branches: [main]
+    paths: ['.github/workflows/debos.yml', 'debos-recipes/*']
```

Comment:
I really don't know why you want to limit the CI like that. Yes, we are saving a 'build' if we only update the README.. but the risk is that we are missing an important change (e.g. a change in script folder). I think it's ok to just test all PRs. We can always limit later, if it's really a problem, no?

Reply:
Sure, this is to be conservative in not triggering things needlessly. We'll be adding a lot of downstream tests over time, and we might build other artifacts from this repo, but one could argue this is premature optimization.

Reply:
Yeah, we can limit it later if needed.

Reply:
I've removed the paths in both event triggers and updated comments; a build with an arm64 self-hosted runner also just passed before this change